app/controllers/tickets_controller.rb

```diff
@@ -5,7 +5,11 @@ class TicketsController < ApplicationController
   authorize_resource :conference_registrations, class: Registration
   before_filter :check_load_resource, only: :index
 
-  def index; end
+  def index
```

Comment:
So basically we need to avoid 2 cases here:

1. User buying more than 1 registration ticket in a single purchase. This case is avoided by setting max limit on registration tickets.

2. User buying 1 registration tickets in multiple ticket purchases so actually having more than one registration tickets. To avoid this we check if the user already has a registration ticket then we don't render registration ticket on the index page thus not giving the ability to the user to buy any more registration ticket.